spring-integration-core/src/main/java/org/springframework/integration/dsl/IntegrationFlows.java

```diff
@@ -51,7 +51,10 @@
  * @since 5.0
  *
  * @see org.springframework.integration.dsl.context.IntegrationFlowBeanPostProcessor
+ *
+ * @deprecated Since version 6.0, this factory is not recommended to use as it shall be removed in the following versions. Use fluent API methods straight from {@link IntegrationFlow} interface.
  */
+@Deprecated(since = "6.0", forRemoval = true)
```

Comment:
You know when I deprecate something, I prefer to delegate to the new API, so when we change something in that API, the deprecated one is going to be affected automatically.